### ISSUE TITLE
rust bitflags error.

### ISSUE THREAD

**Issue by oksangman** [focus-creative-games/luban#155]
I tried testing with the [MiniTemplate ](https://github.com/focus-creative-games/luban_examples/tree/main/MiniTemplate) from the examples to use luban in Rust, but I encountered an error.

## test.rs file
![image](https://github.com/focus-creative-games/luban/assets/579523/7134f784-8638-4c21-82f0-fe26783a98a7)

## Error message
```
error[E0425]: cannot find value `WRITE` in this scope
  --> cfg\src\test.rs:20:28
   |
20 |         const READ_WRITE = WRITE|READ;
   |                            ^^^^^ not found in this scope

error[E0425]: cannot find value `READ` in this scope
  --> cfg\src\test.rs:20:34
   |
20 |         const READ_WRITE = WRITE|READ;
   |                                  ^^^^ not found in this scope

```

## gen.bat file
```batch
dotnet %LUBAN_DLL% ^
    -t server ^
    -d json ^
	-c rust-json ^
    --conf %CONF_ROOT%\luban.conf ^
    -x outputDataDir=out_s_data ^
	-x outputCodeDir=out_s_code
```

**Comment by pirunxi** [comment]
thanks, we will fix it soon.

**Comment by pirunxi** [comment]
we fix it in v2.11.0.
